cli/src/wasm.rs

```diff
@@ -83,6 +83,8 @@ pub fn compile_language_to_wasm(language_dir: &Path, force_docker: bool) -> Resu
         "-s",
         "NODEJS_CATCH_EXIT=0",
         "-s",
+        "NODEJS_CATCH_REJECTION=0",
```

Comment:
After enabling assertions, my first error was the generated tree-sitter grammar's WASM catching a rejection.  Sadly, I have not been able to recreate this for a screenshot.

[This flag (which disables that behavior) is already used in `script/build-wasm`](https://github.com/tree-sitter/tree-sitter/blob/cabdcfb41d780c760a2d0bea32f743b7d9ac8e4e/script/build-wasm#L89)